.github/semantic.yml

```diff
@@ -0,0 +1,26 @@
+# Always validate the PR title AND all the commits
+titleAndCommits: true
+
+# By default types specified in commitizen/conventional-commit-types is used.
+# See: https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json
+# You can override the valid types
+types:
```

Comment:
- The prefixes all need to be in this list? Do we need to be that restrictive?
- If we keep the list, let's alphabetize it
- Does the bot message have a link to this list when it fails?

Reply:
> * The prefixes all need to be in this list? Do we need to be that restrictive?

Well it looks like there is a hardcoded list so we can only add new things to it.

> * If we keep the list, let's alphabetize it

Ok

> * Does the bot message have a link to this list when it fails?

No, it redirects to its repo.